sources/install/package_network.sh

```diff
@@ -37,7 +38,8 @@ function install_network_apt_tools() {
     add-test-command "hydra -h |& grep 'more command line options'" # Login scanner
     add-test-command "mariadb --version"                            # Mariadb client
     add-test-command "redis-cli --version"                          # Redis protocol
-
+    add-test-command "mitmproxy --help"                             # MITMProxy
```

Comment:
This test command returns 1.
For the tests to work, it must return 0. Here is a command that works: `mitmproxy --version`

```bash
> mitmproxy --help

Filters:
  See help in mitmproxy for filter expression syntax.

  --intercept FILTER    Intercept filter expression.
  --view-filter FILTER  Limit the view to matching flows.
```

```bash
> echo $?

1
```

Reply:
I correct the check command !